raft/progress.go

```diff
@@ -283,3 +286,44 @@ func (in *inflights) reset() {
 	in.count = 0
 	in.start = 0
 }
+
+// prs tracks the currently active configuration and the information known about
+// the nodes and learners in it. In particular, it tracks the match index for
+// each peer which in turn allows reasoning about the committed index.
+type prs struct {
```

Comment:
Can we give the type a better name? Maybe `progressTracker`?

Reply:
Done.